Unity-Technologies/Graphics
# [Xpipeline]Improvements on Volume Additional Properties

### Purpose of this PR
- Volume components without custom editor and parameters with `[AdditionalProperty]` were not being handled by their default editor
- Removing the `BeginAdditionalProperty` and `EndAdditionalProperty` of custom inspectors as this is handled by the default/base `VolumeComponentEditor` class, also this behaviour has been changed to a `IDisposable`
- Now, the base method `PropertyField` returns if a property has been drawn, this allow inherited classes to simply add check based on other properties(for example `DOFEditor`)
- Fix correct alignment of Additional Properties.

---
### Testing status
- Check that the properties that are defined as additional are shown/hidden when they 'Show Additional Properties' is changed.
- Check that the additional properties are correctly right-aligned to non additional properties
